Makefile

```diff
@@ -37,5 +37,11 @@ cover:
 	@echo "+ $@"
 	@go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' $(shell go list ${PKG}/... | grep -v vendor) | xargs -L 1 sh -c
 
+coveralls:
+	@echo "+ $@"
+	@go get github.com/mattn/goveralls
+# Depends on $COVERALLS_TOKEN (in config.yaml for automated runs)
+	@goveralls
```

Comment:
FYI for comparison, this is what we use at other places: https://github.com/kubernetes/ingress-nginx/blob/b68feb9c590e2c11bdb098549f3208f80029daf7/Makefile#L151.
Dont we need to pass the repotoken?

Reply:
https://github.com/mattn/goveralls#usage

> You can set the environment variable $COVERALLS_TOKEN to your token so you do not have to specify it at each invocation.

Reply:
`COVERALLS_TOKEN` is being set on the prow-job-container's env in the test-infra PR

Reply:
It looks like the difference is that ingress-nginx runs "go test" separately, and in doing so, also filters out "test/e2e". Should we do that?

Reply:
Yes we can filter it out, but not really a blocker for this PR. We can also filter it in the PR that adds that directory.